### PR TITLE
Update microsoft-365-business-faqs.md

### PR DESCRIPTION
**Pull request by msbemba**
Correct Compatibility as this changed recently.

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/2390

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/business/support/microsoft-365-business-faqs.md [hunk 1]
```diff
@@ -122,15 +122,15 @@ Microsoft 365 Business Premium is available to all partners and customers where
...
 
 ### Does Microsoft 365 Business Premium include the full capabilities of Microsoft Intune? 
-Yes, Microsoft 365 Business Premium subscribers are licensed to use full Intune capabilities for iOS, Android, MacOS, and other cross-platform device management. Features not available in the simplified management console in Microsoft 365 Business Premium, like third-party app management and configuration of WiFi profiles, VPN certificates, can be managed in the full Intune console. 
+Yes, Microsoft 365 Business Premium subscribers are licensed to use full Intune capabilities for iOS, Android, macOS, and other cross-platform device management. Features not available in the simplified management console in Microsoft 365 Business Premium, like third-party app management and configuration of WiFi profiles, VPN certificates, can be managed in the full Intune console.
```
Comment (JohanFreelancer9) [inline microsoft-365/business/support/microsoft-365-business-faqs.md]:
```suggestion
Yes, Microsoft 365 Business Premium subscribers are licensed to use full Intune capabilities for iOS, Android, macOS, and other cross-platform device management. Features not available in the simplified management console in Microsoft 365 Business Premium, like third-party app management, configuration of Wi-Fi profiles, and VPN certificates, can be managed in the full Intune console. 
```

File: microsoft-365/business/support/microsoft-365-business-faqs.md [hunk 2]
```diff
@@ -139,21 +139,21 @@ Yes, Azure AD Premium P1 is included with Microsoft Business Premium.
...
+All the add-ons that can be added to Microsoft 365 Business Standard can be added to Microsoft 365 Business Premium. This means that you can purchase Office 365 Cloud App Security, Advanced Compliance, Threat Intelligence, MyAnalytics, Power BI Pro, and Audio Conferencing. 
  
 ### Can I add Phone System and Calling Plans to Microsoft 365 Business Premium? 
 No, Phone System and Calling Plan are reserved for customers who have more advanced needs. Customers who require these capabilities should look at Microsoft 365 Enterprise offerings. 
  
-### Can Microsoft 365 Business Premium customers use Windows Defender Advanced Threat Protection? 
-No, customers that require Windows Defender Advanced Threat Protection need either Windows 10 Enterprise E5 or Microsoft 365 Enterprise E5. 
+### Can Microsoft 365 Business Premium customers use  Microsoft Defender Advanced Threat Protection? 
+Yes, Microsoft Defender Advanced Threat Protection is available as an Add-on to Microsoft 365 Business premium customers.
```
Comment (JohanFreelancer9) [inline microsoft-365/business/support/microsoft-365-business-faqs.md]:
```suggestion
Yes, Microsoft Defender Advanced Threat Protection is available as an add-on to Microsoft 365 Business Premium customers.
```

**Review by JohanFreelancer9 — APPROVED** [review]
Copy review.

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor edits.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Sirkkuw Please merge this PR. Thanks.

**Review by Sirkkuw — APPROVED** [review]

**Pull request merged**